StorageTests/TestSQLiteHistory.swift

```diff
@@ -99,9 +106,8 @@ class BaseHistoricalBrowserTable {
 //
 // These tests snapshot the table creation code at each of these points.
 
-class BrowserTableV6: BaseHistoricalBrowserTable {
-    var name: String { return "BROWSER" }
-    var version: Int { return 6 }
+class BrowserSchemaV6: BaseHistoricalBrowserSchema {
```

Comment:
Yeah.. as we discussed before, I plan on going back through here and filling in the gaps in the schema upgrade testing soon. I probably will just capture DB's and add them in as fixtures because it might be cleaner and more representative of real-world code execution.